src/client/daemon.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <unistd.h>
+#include <syslog.h>
+#include <string.h>
+
+#include "clipboard.h"
+#include "config.h"
+
+
+class Daemon
+{
+  public:
+    Daemon();
```

Comment:
You should put comments on each of the public function, detailing what they do, what arguments they take in, what arguments they return, etc